pkg/client/automation/client.go

```diff
@@ -267,3 +280,18 @@ func rmIDField(data *[]byte) error {
 	}
 	return nil
 }
+
+func get(a Client, resourceType ResourceType, offset int) (rest.Response, error) {
+
+	params := url.Values{}
+	params.Add("offset", strconv.Itoa(offset))
+
+	u, _ := url.ParseRequestURI(a.url)
+	u.Path = a.resources[resourceType].Path
```

Comment:
I'm not 100% sure, but I think this could cause issues in some Managed or other proxy cases where the configured environment URL isn't just a host. 

If I'm not mistaken this would result in the created URL containing only the `host` part of the `client.url` value - possibly losing any path that already contains. 

I think just concatenating the URL and resource Path as was done before or via path.Join would be safer

Reply:
The new logic has been implemented. It is now located in the `internal` package. Please take a look!